frame/support/src/traits/messages.rs

```diff
@@ -85,13 +93,20 @@ pub trait ServiceQueues {
 	fn service_queues(weight_limit: Weight) -> Weight;
 
 	/// Executes a message that could not be executed by [`Self::service_queues()`] because it was
-	/// temporarily overweight.
+	/// permanently overweight.
```

Comment:
can you explain this change?
I think it supposed to handle temporarily overweights, otherwise it will fail again as overweight